### PR TITLE
add 'tiup cluster import’ in tidb-using-tiup

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/6007
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Liuxiaozhen12** [comment]
translation from-docs-cn

File: upgrade-tidb-using-tiup.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ If you have deployed the TiDB cluster using TiDB Ansible, you can use TiUP to im
 - Support upgrading the versions of TiDB Binlog, TiCDC, TiFlash, and other components.
 - Before you upgrade from v2.0.6 or earlier to v4.0.0 or later, you must make sure that no DDL operations are running in the cluster, especially the `Add Index` operation that is time-consuming. Perform the upgrade after all DDL operations are completed.
 - Starting from v2.1, TiDB enables parallel DDL. Therefore, clusters **older than v2.0.1** cannot be upgraded to v4.0.0 or later via a direct rolling upgrade. Instead, you can choose one of the following solutions:
-    - Upgrade directly from TiDB v2.0.1 or earlier to v4.0.0 or later in planned downtime
-    - Perform a rolling upgrade from the current version to v2.0.1 or a later 2.0 version, then perform another rolling upgrade to v4.0.0 or later
+    - Upgrade directly from TiDB v2.0.1 or earlier to v4.0.0 or later in planned downtime. Then use `tiup cluster import` command provided by TiUP Cluster to import configurations and use TiUP as the management tool at the same time.
```
Comment (TomShawn) [inline upgrade-tidb-using-tiup.md:29]: It is not at the same time. We must first import the configuration and then TiUP is the management tool.
```suggestion
    - Upgrade directly from TiDB v2.0.1 or earlier to v4.0.0 or later in planned downtime. Then use `tiup cluster import` command provided by TiUP Cluster to import configurations and use TiUP as the management tool.
```

File: upgrade-tidb-using-tiup.md [hunk 2]
```diff
@@ -26,8 +26,8 @@ If you have deployed the TiDB cluster using TiDB Ansible, you can use TiUP to im
...
+    - Perform a rolling upgrade from the current version to v2.0.1 or a later 2.0 version, then perform another rolling upgrade to v4.0.0 or later. Then use `tiup cluster import` command provided by TiUP Cluster to import configurations and use TiUP as the management tool at the same time.
```
Comment (TomShawn) [inline upgrade-tidb-using-tiup.md:30]:
```suggestion
    - Perform a rolling upgrade from the current version to v2.0.1 or a later 2.0 version, then perform another rolling upgrade to v4.0.0 or later. Then use `tiup cluster import` command provided by TiUP Cluster to import configurations and use TiUP as the management tool.
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**